content/getting-started/multiDeviceMultiSig/_index.en.md

```diff
@@ -0,0 +1,187 @@
+
+<img src = desktopSelectKey.png
+    alt = "Assign Key Desktop (Screenshot)">
+
+11. You should now see the original chat screen you created in step 2. This is where you can find the pending wallet configuration when you begin adding your keys on the next device.<br/><br/>
+
+<img src = desktopConfigDone.png
+    alt = "Chat screen pending keys (Screenshot)">
+
+{{< notice note >}}
+You do not need to continue to the other tabs if you were able to add all the keys to your wallet (number 9). If you want to add a key created from another device, continue to the next tab.
+{{< /notice >}}
+
+{{< /tab >}}
+
+{{< tab "Laptop" >}}
+
+{{< notice note >}}
+If you did not have all your keys available for the initial setup, you can continue from another device to continue the setup process. It is assumed that you have started to create a wallet. If you still have a pending configuration notification in your chat screen you can continue on your laptop here. If you do not, you should refer to the desktop tab to start a configuration.
+{{< /notice >}}
+
+1. Sign into your laptop and open Nunchuk. If you need help installing, refer to the install documentation found <u> [here](/getting-started/createsoftwarekey/) </u>
+<br/><br/>
+2. Navigate to the chat screen. There you should see the pending wallet notification. Click the "View config" button.<br/><br/>
+
+<img src = laptopConfigNote.png
+    alt = "pending keys config note (Screenshot)">
+
+3. Here you see the number of current keys assigned among other information about the wallet. Click the "Continue to assign keys" button.<br/><br/>
+
+<img src = laptopWalletConfig.png
+    alt = "wallet configuration (Screenshot)">
+
+4. You can now assign the key stored locally on your laptop. Notice that the other two keys are considered foreign and your local key is recognized as a "SOFTWARE" key.<br/><br/>
+
+<img src = laptopAssignKey.png
+    alt = "assign key configuration (Screenshot)">
+
+5. After assigning your key, click the "Continue" button. You should now be returned to the chat screen and see that your config notification is now only pending 1 key.<br/><br/>
+
+<img src = laptopPendingKeys.png
+    alt = "pending keys (Screenshot)">
+
+6. Continue to your next device!
+
+{{< /tab >}}
+
+{{< tab "Mobile" >}}
+
+{{< notice note >}}
+If you did not have all your keys available for the initial setup, you can continue from another device to continue the setup process. It is assumed that you have started to create a wallet. If you still have a pending configuration notification in your chat screen you can continue on your laptop here. If you do not, you should refer to the desktop tab to start a configuration.
+{{< /notice >}}
+
+1. Open your Nunchuk mobile app and navigate to the chat screen. You should see your wallet config notification here waiting for the last key!<br/><br/>
+
+<img src = mobileChatScreen.png
```

Comment:
the mobile screenshots don't look legible for some reason...

<img width="283" alt="Screen Shot 2022-03-14 at 2 59 00 PM" src="https://user-images.githubusercontent.com/4769925/158268400-24f71228-9eeb-40e3-ada6-34844cc3e8ff.png">

Reply:
In the website the mobile screenshots were very large... So I resized them using "paint" lol. Do you have a better idea on how to do that? I can redo them using your technique.

Reply:
@DanielG9d9 I usually use online image resize tools. like this one https://resizeimage.net/

Reply:
That tool worked better! Updated.